src/book/02-system/02-subsystems/06-behaviour.mdx

```diff
@@ -57,7 +57,99 @@ The module connects to the PS3 controller using bluetooth (or USB) and reacts ba
 | SQUARE                    | Toggles locking of the head.               |
 | R2                        | Right kick.                                |
 
-To use this module, build the [PS3Walk role](https://github.com/NUbots/NUbots/blob/main/roles/ps3walk.role) and run the binary. Plug the PS3 controller into the robot using a USB cable. When the controller is connected to the robot, it will be paired, and you can remove the cable.
+To use this module, build the [PS3Walk role](https://github.com/NUbots/NUbots/blob/main/roles/ps3walk.role) and install it on the robot. Ensure the PS3 controller is paired and connected to the right robot. Then run the `./ps3walk` on the robot to begin controlling it.
```

Comment:
For someone not familiar with this, I feel like the different steps required can get confusing. And extra line or two explaining this might be good.

```suggestion
To use this module, build the [PS3Walk role](https://github.com/NUbots/NUbots/blob/main/roles/ps3walk.role) and install it on the robot. Ensure the PS3 controller is paired and connected to the right robot. You can setup bluetooth on the robot and pair the controller using the steps below. Once these have been run once, the robot should automatically connect to the controller when both the robot and controller are powered. When you are ready, run `./ps3walk` on the robot to begin controlling it.
```